packages/core/src/loop/index.tsx

```diff
@@ -53,10 +53,11 @@ export const Loop: React.FC<LoopProps> = ({
 						key={`loop-${i}`}
 						durationInFrames={durationInFrames}
 						from={i * durationInFrames}
-						layout={layout}
 						name={name}
 						showLoopTimesInTimeline={actualTimes}
 						showInTimeline={i === 0}
+						layout={props.layout}
+						{...(props.layout==='absolute-fill' && {style: props.style})}
```

Comment:
while this is more explicit it can be changed to as follows based on your preference
```suggestion
						{...props}
```

Reply:
Thanks! I like it but we also need to handle the case where layout is undefined (the default case where it defaults to `absolute-fill`). Fixed that now!